Missed @api.multi

FullBringEc opened the pull request:
TypeError: assign_outstanding_credit() takes exactly 2 arguments (5 given)

FullBringEc committed:
Missed @api.multi

TypeError: assign_outstanding_credit() takes exactly 2 arguments (5 given)

moylop260 commented on account_outstanding_payment/models/account_invoice.py:
```diff
@@ -7,8 +7,9 @@
+import logging
 
-
+logging.basicConfig(level=logging.DEBUG)
```
This change is out-of-scope of the description of the PR

FullBringEc replied:
Yes, sorry I was testing and forgot to delete that line.

FullBringEc committed:
Missed @api.multi

Fix to TypeError: assign_outstanding_credit() takes exactly 2 arguments (5 given)
